modules/assim.sequential/R/Analysis_sda.R

```diff
@@ -325,12 +367,11 @@ GEF<-function(settings, Forecast, Observed, H, extraArg, nitr=50000, nburnin=100
   
   if(any(grep(names(data_available),pattern = 'ALR'))){
     
-    # browser()
     which_fcomp <- grep(names(data_available),pattern = 'ALR')
     X_fcomp_start <- which_fcomp[1]
     X_fcomp_end <- which_fcomp[length(which_fcomp)]
     X_fcomp_model <- grep(colnames(X),pattern = 'AGB.pft')
```

Comment:
Do you think `*.pft.*` would work for your case/project instead of `AGB.pft` ? `*.pft.*` makes things much easier for ED state variables

Reply:
Yes! Thanks. I didn't know about the double *